json/test_simdjson.cpp

```diff
@@ -41,11 +40,9 @@ bool on_demand::run(const padded_string &json) {
   count = 0;
   auto doc = parser.iterate(json);
   for (ondemand::object point_object : doc["coordinates"]) {
```

Comment:
You can use auto here instead of ondemand::object, now!

Reply:
+1